python/lsst/ts/externalscripts/auxtel/build_pointing_model.py

```diff
@@ -241,13 +235,19 @@ def estimated_average_slew_time(self):
 
     async def arun(self, checkpoint_active=False):
 
+        self.log.info("Reset pointing and hexapod offsets.")
```

Comment:
Resetting (otherwise it sounds like the user should be doing it).
Consider:
"Resetting pointing and hexapod x and y offsets."